### PR TITLE
Kumarabd/feature/helm

### LINKED ISSUE #2837: [Meshery] Helm chart access to be updated
#### Description
Currently meshery helm chart does not have the permissions to do operations on MeshSync custom resources.

#### Expected Behavior
The helm chart needs to be modified, to grant access to these custom resources. The access levels can be picked up from here - https://github.com/layer5io/meshery-operator/blob/master/config/manifests/default.yaml#L199-L320.

#### Screenshots/Logs
```
time="2021-05-05T06:29:13Z" level=error msg="Meshery kubernetes client not initialized.brokers.meshery.layer5.io \"meshery-broker\" is forbidden: User \"system:serviceaccount:meshery:meshery-service-account\" cannot get resource \"brokers\" in API group \"meshery.layer5.io\" in the namespace \"meshery\"" app=meshery code=test_code probable-cause=None severity=1 short-description=None suggested-remediation=None
```

#### Environment:
- Meshery Version: any
- Kubernetes Version: any
- Host OS: any
- Browser: any

### PR DESCRIPTION
**Pull request by kumarabd**
**Description**

- This PR fixes #2837 
- The ports are remapped to listen to `9081`

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Anita-ihuman** [comment]
> @Anita-ihuman fyi. Please note this with respect to your last update to the port number for the localhost endpoint after installing Meshery with helm.

I don't understand, Please can you explain further?

File: ui/components/MesheryPerformance/index.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ function generatePerformanceProfile(data) {
 const loadGenerators = [
   "fortio",
   "wrk2",
-  // 'nighthawk',
+ 'nighthawk',
```
Comment (leecalcote) [inline ui/components/MesheryPerformance/index.js:91]:
```suggestion
  'nighthawk',

```

Comment (leecalcote) [inline ui/components/MesheryPerformance/index.js]:
```suggestion
  'nighthawk',

```

**Review by leecalcote — CHANGES_REQUESTED** [review]
@kumarabd - 

```
go: github.com/layer5io/nighthawk-go@v0.1.6: parsing /Users/abishekk/Documents/layer5/getnighthawk/go.mod: open /Users/abishekk/Documents/layer5/getnighthawk/go.mod: no such file or directory
```

File: Dockerfile [hunk 2]
```diff
@@ -42,6 +42,14 @@ COPY --from=provider-ui /out /app/provider-ui/out
 COPY --from=wrk2 /wrk2 /app/cmd/wrk2
 COPY --from=wrk2 /wrk2/wrk /usr/local/bin
 COPY --from=nighthawk /nighthawk-go/apinighthawk/bin /usr/local/bin
+RUN curl -Lo nighthawk_service-ubuntu-amd64-v1.0.1.tar.gz https://github.com/layer5io/getnighthawk/releases/download/v1.0.1/nighthawk_service-ubuntu-amd64-v1.0.1.tar.gz \
```
Comment (leecalcote) [inline Dockerfile:45]: Don't hardcode the release.

**Comment by kumarabd** [comment]
> > @Anita-ihuman fyi. Please note this with respect to your last update to the port number for the localhost endpoint after installing Meshery with helm.
> 
> I don't understand, Please can you explain further?

It means meshery will be accessible from `localhost:9081` instead of `localhost:8080`

**Comment by Anita-ihuman** [comment]
Alright then, will make the change.

**Comment by kumarabd** [comment]
@ramrodo @Aisuko Any idea why the releaser chart is failing?

**Comment by Aisuko** [comment]
> @ramrodo @Aisuko Any idea why the releaser chart is failing?

I'm working on this, please kindly manually push the latest chart to the meshery.io

**Comment by kumarabd** [comment]
@Aisuko Will you be able to link the issue that you are working on in here?

**Comment by Aisuko** [comment]
issue https://github.com/layer5io/meshery/pull/2700

**Pull request closed without merging**